build.assets/docs.dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/gravitational/mkdocs-base:1.0.3
+FROM quay.io/gravitational/mkdocs-base:1.0.3-alpha
```

Comment:
Ideally we should push this up to `1.0.4` I think rather than using an `alpha` tag? If this is just for testing it's ok but the production tag should be a proper semver.

Reply:
I was mainly using MkDocs version number, as the main indicator of the docs image. Actually I'll update it 1.0.4 and include this update, I just noticed I didn't pick the newest release https://github.com/mkdocs/mkdocs/releases

Reply:
Just updated ⭐️